Examples/Sample_xServiceSet.ps1

```diff
@@ -1,20 +1,27 @@
-﻿# Starts a set of existing stopped services
+﻿<#
+    .SYNOPSIS
+        Sets the services with the given names to start under the built-in account LocalService.
 
-Configuration xServiceSetExample
+    .PARAMETER ServiceNames
+        The names of the services to set.
+#>
+Configuration xServiceSetBuiltInAccountExample
 {
-    param (
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
         [String[]]
-        $StoppedServiceNames
+        $ServiceNames
```

Comment:
@TravisEz13 I renamed this file to Sample_xServiceSet_BuiltInAccount below. Sorry forgot to remove the old one.
Is the file rename/refocus what you want here? Or are you suggesting removing all ServiceSet examples entirely?